db/migrate/20160321160404_add_middleware_event_streams.rb

```diff
@@ -0,0 +1,7 @@
+class AddMiddlewareEventStreams < ActiveRecord::Migration[5.0]
+  def change
+    add_column :event_streams, :middleware_type, :string   # relevant type (entity class name)
+    add_column :event_streams, :middleware_id, :bigint     # MIQ entity id
```

Comment:
@abonas @durandom, can one of you give me an example of the preferred style?  I modeled this after code that I saw, I'm happy to use a different approach if you give me an example of how better to add rows to the table.